en/docs/manage/api-proxies.md

```diff
@@ -1,4 +1,4 @@
 # API Proxies
 
 !!! info
-    This section will describe API proxies, its concepts, and its usage in Choreo.
+    This section will describe the API proxy, its concepts, and its usage in Choreo.
```

Comment:
Since the heading is plural, should we make it singular in the description? Also, is it correct to make API Proxy singular?

Reply:
In that case, let's change it to "This section will describe "API proxies, and their concepts and usage in Choreo."?

We can't use "its" with plural.